Submissions/kavi-12/#task-1/csvtojson.py

```diff
@@ -0,0 +1,22 @@
+import csv, json
+import pandas as pd
+import os
+#os.chdir("..//..")
+csv_path = "../../../Task_1/movies.csv"
+#print(csv_path)
+json_path = "comedy_movies.json"
+
+data = {}
+with open(csv_path,  encoding='utf-8') as csvfile:
+    csvReader = csv.DictReader(csvfile)
+    for csvrow in csvReader:
+        
+        obj = {
+            "title": csvrow["title"],
+            "genres": csvrow["genres"]
+            
```

Comment:
In the json file created movieId is missing so make changes in your code accordingly.